infra/label_validation_agent_step_functions/handlers/aggregate_results.py

```
@@ -0,0 +1,259 @@
+        for page in pages:
+            for obj in page.get("Contents", []):
+                key = obj["Key"]
+                if key.endswith(".json"):
+                    total_objects += 1
+                    try:
+                        response = s3.get_object(Bucket=batch_bucket, Key=key)
+                        result = json.loads(response["Body"].read().decode("utf-8"))
+                        results.append(result)
+                    except Exception as e:
+                        failed_keys.append(key)
+                        logger.exception(f"Failed to load result from {key}: {e}")
```

Ruff (0.14.8)

Redundant exception object included in `logging.exception` call

(TRY401)